cfgov/jinja2/v1/_includes/organisms/full-width-text.html

```diff
@@ -29,12 +29,14 @@
                 {% set media = image(block.value, 'original') %}
                 <img class="m-full-width-media_image" src="{{ media.url }}" alt="{{ media.alt }}">
             </div>
-        {% elif 'image_inset' in block.block_type %}
-            {{ block | safe }}
-        {% else %}
+        {% elif block.block_type in ['quote', 'cta', 'related-links'] %}
             <div class="m-inset">
                 {{ block | safe }}
             </div>
+        {% elif 'reusable_text' == block.block_type %}
```

Comment:
Right now there's no custom template for this snippet, since I didn't create a Wagtail block for it the way there exists `MainContactInfo` for the contact snippet.  This seems like the most straightforward way to implement this, but I'm open to suggestions as ideally we'd move away from having this type of logic in the template.